examples/webgl_loader_viewer.html

```diff
@@ -0,0 +1,410 @@
+				var percentComplete = xhr.loaded / xhr.total * 100;
+				console.log( Math.round(percentComplete, 2) + '% downloaded');
+			}
+		};
+
+		var onError = function ( xhr ) {
+		};
+
+		var manager = new THREE.LoadingManager();
+		manager.onProgress = function ( item, loaded, total ) {
+
+			console.log( item, loaded, total );
+
+		};
+
+		var loader = selectLoader(fileType, manager);
+
+		if (fileType === "dae" )
+		{
+			loader.options.convertUpAxis = true;
+		}
+		else if (fileType === "pcd"){
+			//do something to invert upside down
+		}
+
+		clearScene();
+
+		//to do: improvement - setupLights() improve to use better lights
+		setupLights();
+
+		//to do: improvement - loader.load() note: sea loader does not use onload function - loader.load 2nd param
+		loader.load( objPath , function ( myScene ) {
+
+			debugPrint(objPath+"<br>");
+			OnLoadScene(myScene, fileType, fileName);
+			resetCamera();
+
+		}, onProgress, onError );
+
+		//to do: improvement - post loader.load() fixes - handle camera for these files automaticaly
+		if (fileName==="mascot.tjs.sea")
+		{
+			camera.position.z = 2000;
+		}
+		else if (fileName==="house.wrl")
+		{
+			camera.position.z = 30;
+		}
+		else if (fileName==="kawada-hironx.dae"){
+			camera.position.z = 3;
+		}
+		else if (fileName==="duck.gltf"){
+			camera.position.z = 6;
+		}
+		else if (fileName==="Zaghetto.pcd")
+		{
+			//to do: improvement - Zaghetto.pcd need to flip pcd model upside down as needed
+			//to do: improvement - Zaghetto.pcd find correct camera settings
+		}
+	}
+
+	function init() {
+
+		camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 1, 2000 );
+		//camera.position.z = 100;
+		camera.position.z = 250;
+
+		//controls = new THREE.TrackballControls( camera );
+
+		// scene
+
+		scene = new THREE.Scene();
+
+		// texture
+
+		var texture = new THREE.Texture();
+
+		var material = new THREE.MeshBasicMaterial( { color: 'red' } );
+
+		// model
+		var model = 'models/babylon/skull.babylon';
+		loadModel(model);
+
+		//
+
+		renderer = new THREE.WebGLRenderer();
+		renderer.setPixelRatio( window.devicePixelRatio );
+		renderer.setSize( window.innerWidth, window.innerHeight );
+		document.body.appendChild( renderer.domElement );
+
+		//
+
+		controls = new THREE.OrbitControls( camera, renderer.domElement );
+
+		window.addEventListener( 'resize', onWindowResize, false );
+
+	}
+
+	function OnLoadScene(fileScene, objType, objName)
+	{
+
+		if (objType === "dae" || objType === "gltf" || objType === "sea")
+		{
+			fileScene = fileScene.scene;
+			//scene.scale.x = scene.scale.y = scene.scale.z = 0.05;
+		}
+
+		debugPrint("root children:" + fileScene.children.length+"<br>");
+		var space ="";
+		var childcnt = 0;
+		fileScene.traverse( function ( object ) {
+
+			//to do: bug - fileScene.traverse() dae error on traversing last child?
+
+			//to do: improvement - fileScene.traverse() handle nested groups?
+			if ( object.constructor.name === "Group" ) {
```

Comment:
@jsidd this may cause problems with three.min.js

You should use object.type === "Group" instead.
See #10105